providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/executors/kubernetes_executor.py

```diff
@@ -385,6 +385,7 @@ def sync(self) -> None:
                     if (
                         (str(e.status) == "403" and "exceeded quota" in message)
                         or (str(e.status) == "409" and "object has been modified" in message)
+                        or str(e.status) == "500"
```

Comment:
Is there a particular error message you see along with that? Can make the check more robust that way